l3kernel/l3regex.dtx

```diff
@@ -7452,7 +7452,7 @@
 %   token lists.  Then build the automaton with |#1|, without submatch
 %   tracking, and aiming for a single match.  Then start matching by
 %   setting up a few variables like for any regex matching like
```

Comment:
Too many "like"s !! 
I got lost, this clearly needs rewording to be understandable!